mod/admin.php

```diff
@@ -914,7 +914,7 @@ function admin_page_summary(App $a)
 	// Legacy config file warning
 	if (file_exists('.htconfig.php')) {
 		$showwarning = true;
-		$warningtext[] = L10n::t('Friendica\'s configuration now is stored in config/local.ini.php, please copy config/local-sample.ini.php and move your config from <code>.htconfig.php</code>. See <a href="%s">the Config help page</a> for help with the transition.', $a->getBaseURL() . '/help/Config');
+		$warningtext[] = L10n::t('Friendica\'s configuration now is stored in config/local.config.php, please copy config/local-sample.config.php and move your config from <code>.htconfig.php</code>. See <a href="%s">the Config help page</a> for help with the transition.', $a->getBaseURL() . '/help/Config');
```

Comment:
Could you please regenerate the master messages.po file and include it into the PR so that the translation system can pick up all the changes you have introduced.

Reply:
The `doc/translations.md` still doesn't indicate how to do that...

Reply:
Oh you are right. But there is still the reference to the `utils/README` file. The person who moved the utils out of the directory seems to have forgotten to update the documentation to reflect the change.

The section starts with

> Xgettext and .po workflow
>
> 1. Run bin/run_xgettext.sh script (on *unix sistems, with GNU xgettext installed)
> 	This script runs xgettext on source tree, extracting strings from L10n::t() and L10n::tt()
>	functions, and creates a util/messages.po file.

I'll try to update the translation documentation before the release.